sefaria/helper/link.py

```diff
@@ -99,7 +99,7 @@ def _save_link(self, tref, base_tref, **kwargs):
             else:
                 tracker.add(self._user, Link, nlink, **kwargs)
         except DuplicateRecordError as e:
-            pass
+            print e.message
```

Comment:
In production code, we use `logger.warning()` or `logger.error()`

Reply:
But I'm worried about this being too chatty and filling up the logs.  Duplicate links come up all the time.